scene/main/node.cpp

```diff
@@ -1736,11 +1736,11 @@ Node *Node::get_node_or_null(const NodePath &p_path) const {
 		StringName name = p_path.get_name(i);
 		Node *next = nullptr;
 
-		if (name == SceneStringName(dot)) { // .
+		if (name == SNAME(".")) {
 
 			next = current;
 
-		} else if (name == SceneStringName(doubledot)) { // ..
+		} else if (name == SNAME("..")) {
 
 			if (current == nullptr || !current->data.parent) {
```

Comment:
That seems to be what it took clang-format to realize that those empty lines after `{` are not following the style.

Reply:
I was wondering why the style was wrong.